files/content/coveryourselfinoil/effect.lua

```diff
@@ -5,7 +5,9 @@ if root and root ~= entity_id and GameGetGameEffectCount(root, "WET") > 0 then
 	local controls_comp = EntityGetFirstComponentIncludingDisabled(root,"ControlsComponent")
 	if not controls_comp or ComponentGetValue2(controls_comp, "mButtonDownDown")then return end;
 	if(not character_data_comp)then return end
-	GameAddFlagRun("oiled_up")
+	if EntityHasTag(root, "player_unit") then
```

Comment:
Should it check for polymorphed_player tag too?

Reply:
Good point, will add